scipy/integrate/tests/test__quad_vec.py

```diff
@@ -104,6 +113,25 @@ def test_quad_vec_pool():
         assert_allclose(res, np.pi, rtol=0, atol=1e-4)
 
 
+def _func_with_args(x, a):
+    return x * (x + a) * np.arange(3)
+
+
+def test_quad_vec_pool_args():
+    from multiprocessing.dummy import Pool
```

Comment:
Since it is now used in two test methods:`test_quad_vec_pool_args` and `test_quad_vec_pool`, @tupui should I move this import to the top of the file?

Reply:
Sure